app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -190,7 +190,28 @@ <h2>Upload Sessions</h2>
 
         <h2>Images</h2>
 
-        <p>There are {{ object.images.all.count }} image(s) in this reader study.</p>
+        {% with image_count=object.images.count %}
+            <p>There {{ image_count|pluralize:"is,are" }} {{ image_count }} image{{ image_count|pluralize }} in this reader study.</p>
+        {% endwith %}
+
+        <ul class="list-group list-group-flush">
+            {% for image in object.images.all %}
+                <li class="list-group-item">
+                    <div class="row">
+                        <div class="col-9 mb-1">
+                            <div class="h-100 d-flex justify-content-start align-items-center">
+                                {{ image.name }} ({{ image.shape_without_color|join:"x" }})
+                            </div>
+                        </div>
+                        <div class="col-3 mb-1">
+                            <div class="d-flex justify-content-end align-items-center">
+                                <button class="btn btn-danger remove-image" data-id="{{ image.id }}">Remove</button>
+                            </div>
+                        </div>
+                    </div>
+                </li>
+            {% endfor %}
+        </ul>
```

Comment:
I think that this needs to be moved to a separate page, we have some studies with 1000's of images.